### PR TITLE
[dbquery] JDBC database bridge

### PR DESCRIPTION
**Pull request by lujop**
Introduce a Bridge to allow JDBC queries.

Implementation is nearly finished and it's working at my home device but needs some cleaning and documentation

### CODE REVIEW

**Comment by lujop** [comment]
JDBC implementation is finished and ready for review

File: bundles/org.openhab.binding.dbquery/README.md [hunk 1]
```diff
@@ -33,6 +33,34 @@ Defines a connection to an Influx2 database and allows creating queries on it.
...
+
+- *PostgreSQL*  (with driver version 42.2.18)
+- *Derby* (with driver version 10.12.1.1)
+- *H2* (with driver version 1.4.191)
+- *HSQLDB* (with driver version 2.3.3)
+- *MariaDB* (with driver version 1.3.5)
+- *MySQL* (with driver version 8.0.26)
+- *Sqlite* (with driver version 3.16.1)
+
+JDBC driver must be installed in your OpenHab server. For convenience, if you only need one driver the addon provides several features with JDBC driver included for each of the previous databases.
```
Comment (jlaur) [inline bundles/org.openhab.binding.dbquery/README.md:58]: Minor, but please reformat to one sentence per line.

Comment (jlaur) [inline bundles/org.openhab.binding.dbquery/README.md:58]:
```suggestion
JDBC driver must be installed on your openHAB server. For convenience, if you only need one driver the addon provides several features with JDBC driver included for each of the previous databases.
```

File: bundles/org.openhab.binding.dbquery/README.md [hunk 2]
```diff
@@ -33,6 +33,34 @@ Defines a connection to an Influx2 database and allows creating queries on it.
...
+If you need more than one driver you must install them manually from OpenHab console.
```
Comment (jlaur) [inline bundles/org.openhab.binding.dbquery/README.md:59]:
```suggestion
If you need more than one driver you must install them manually from the openHAB console. 
```

File: bundles/org.openhab.binding.dbquery/README.md [hunk 3]
```diff
@@ -182,10 +212,12 @@ Example (using Jython script):
 
 ## Examples
 
-### The Simplest case 
+### The Simplest case InfluxDB2
```
Comment (jlaur) [inline bundles/org.openhab.binding.dbquery/README.md:215]: This covers also JDBC now, so why is InfluxDB2 added to chapter name?

File: bundles/org.openhab.binding.dbquery/TODO [hunk 4]
```diff
@@ -0,0 +1 @@
+Si la configuració de la bbdd és buida el item no es connecta pero el log es molt dolent
```
Comment (jlaur) [inline bundles/org.openhab.binding.dbquery/TODO:1]: Please remove this.

**Review by jlaur — CHANGES_REQUESTED** [review]
Thanks for your contribution and patience. I'm sorry you have not received any feedback after several months, and yet I don't have the expertise to assess all parts of this PR, especially the dependencies towards external libraries. I hope a more seasoned maintainer (ping @openhab/add-ons-maintainers) can step in and help. I must say that the code looks pretty good to me though.

**Comment by jlaur** [comment]
@lujop - are you still interested in a review?

**Comment by lsiepel** [comment]
As there has been no activity for over two years, I guess we cannot get this to the finish line. I close this PR for now. If ever needed it can be re-opened again.

**Pull request closed without merging**